pkg/crd/crds.go

```diff
@@ -36,6 +35,8 @@ import (
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
 
+	kfutil "github.com/kubestellar/kubeflex/pkg/util"
```

Comment:
Also here - the same comment, as the kubeconfig.go Why do I need KF util functions in order to apply CRDs on a CP ?